addons/mail/wizard/mail_resend_message_views.xml

```diff
@@ -20,7 +20,9 @@
                             <field name="resend" widget="boolean_toggle"/>
                         </tree>
                     </field>
-                    <p attrs="{'invisible':[('has_cancel', '==', False)]}"> <span class="fa fa-info-circle"/> Caution: It won't be possible to send this mail again to the recipients you did not select.</p>
+                    <div class="alert alert-warning" role="alert" attrs="{'invisible':[('has_cancel', '==', False)]}">
```

Comment:
since you're touching the line, might as well fix the weird operator here ;)
```suggestion
                    <div class="alert alert-warning" role="alert" attrs="{'invisible': [('has_cancel', '=', False)]}">
```

Reply:
I noticed this but didn't touched as It was not required for this change.
I will update this too.

Reply:
> I noticed this but didn't touched as It was not required for this change.
> I will update this too.

Sure, normally we ask devs to not lint things that are not related to the PR, to avoid the noise in the diff and history. But as you're touching those lines the noise is already there, so you can do linting at the same time without problem, especially in master :+1: